PICMI_Python/fields.py

```diff
@@ -14,7 +14,7 @@ class PICMI_ElectromagneticSolver(_ClassWithInit):
     """
     Electromagnetic field solver
       - grid: grid object to be used by the solver (grid object)
-      - method: One of 'Yee', 'CKC', 'Lehe', 'PSTD', 'PSATD', or 'GPSTD' (string)
+      - method: One of 'Yee', 'CKC', 'Lehe', 'PSTD', 'PSATD', 'GPSTD' or 'ECT' (string)
```

Comment:
As on Slack: let's add a short reference to the abbreviations here in the docstring :)